dev_guide/deployments.adoc

```diff
@@ -670,3 +670,39 @@ $ oc scale dc frontend --replicas=3
 
 The number of replicas eventually propagates to the desired and current
 state of the deployment configured by the deployment configuration `frontend`.
+
+
+[[assigning-pods-to-specific-nodes]]
+
+== Assigning Pods to Specific Nodes
+
+You can use node selectors in conjunction with labeled nodes to control pod placement.
+
+ifdef::openshift-enterprise,openshift-origin[]
+[NOTE]
+====
+OpenShift administrators can assign labels link:../install_config/install/advanced_install.html#configuring-node-host-labels[during an advanced installation], or link:../admin_guide/manage_nodes.html#updating-labels-on-nodes[added to a node after installation].
+====
+endif::[]
+
+OpenShift cluster administrators can set the default node selector for your project in order to restrict pod placement to specific nodes. As an OpenShift developer, you can set a node selector on a pod configuration to restrict nodes even further.
+
+To add a node selector when creating a pod, edit the pod configuration, and add the `nodeSelector` value. This can be added to a single pod configuration, or in a pod template:
+
+====
+----
+...
+nodeSelector:
+  disktype: ssd
+...
+----
+====
+
+Pods created with a node selector are assigned to nodes with the specified labels.
+
+The labels specified here are added in conjunction to the labels link:../admin_guide/managing_projects.html#using-node-selectors[added by a cluster administrator]. For example, if a project has the `type=user-node` and `region=east` labels added to a project by the OpenShift administrator, and you add the above `disktype: ssd` label to a pod, the pod will only ever be scheduled on nodes that have all three labels. 
```

Comment:
"added in conjunction to" sounds odd to me at least. How about "are used in conjunction with"?